c2corg_ui/externs/bootstrap-slider.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @fileoverview Externs for bootstrap-slider
+ *
+ * @externs
+ */
+
+
+/**
+ * @constructor
+ */
+function Slider(arg1, arg2) {};
+
+
+/**
+ * @param {Object} arg
```

Comment:
This is the list of slider options:
https://github.com/seiyria/bootstrap-slider#options

Do we really want to spend that much time writing a typedef here?

@ginold Is a file that you have created on your own or have you found it somewhere?